lib/mvi/messages/find_candidate_message.rb

```diff
@@ -45,7 +45,7 @@ def validate_types
           errors.add(:given_names, 'should be an array of Strings')
         end
         errors.add(:family_name, 'should be a String') unless @family_name.is_a?(String)
-        errors.add(:dob, 'should be a Time object') unless @dob.is_a?(Time)
+        errors.add(:birth_date, 'should be a Time object') unless @birth_date.is_a?(Time)
```

Comment:
I don't think this is proper use of model validations. This is not something w would want to return to a user. For example "Time object" implies that the developer building the api should have a concept of what Time object is.